### PR TITLE
consumer schema serialization

### PR DESCRIPTION
**Pull request by Big-Vi**
Related to https://github.com/memphisdev/memphis/issues/879

### CODE REVIEW

File: consumer.go [hunk 1]
```diff
@@ -88,6 +92,56 @@ func (m *Msg) Data() []byte {
 	return m.msg.Data
 }
 
+// Msg.DataDeserialized - get message's deserialized data.
+func (m *Msg) DataDeserialized() (any, error) {
```
Comment (Big-Vi) [inline consumer.go:96]: Created a new function here to retrieve deserialized data. I would happy to change the name or change to different appoach to  retrieve deserialized data.

**Comment by idanasulinmemphis** [comment]
Hi @Big-Vi huge. We will review and update ASAP. Once approved will be great to add to the other sdks as well

**Comment by Big-Vi** [comment]
> Hi @Big-Vi huge. We will review and update ASAP. Once approved will be great to add to the other sdks as well

Yes. I'm working on Python and JS SDKs. I'll do PR in next couple of days.

**Comment by idanasulinmemphis** [comment]
@Big-Vi this is amazing

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**